lib/subscriptionClient.ts

```diff
@@ -234,4 +235,60 @@ export class SubscriptionClient extends Client {
     return this._context.managementClient!.receiveDeferredMessages(sequenceNumbers,
       this.receiveMode);
   }
+
+  //#region topic-filters
+
+  /**
+   * Get all the rules associated with the subscription
+   */
+  async getRules(): Promise<RuleDescription[]> {
+    return this._context.managementClient!.getRules();
+  }
+
+  /**
+   * Removes the rule on the subscription identified by the given rule name.
+   * @param ruleName
+   */
+  async removeRule(ruleName: string): Promise<void> {
+    return this._context.managementClient!.removeRule(ruleName);
+  }
+
+  /**
+   * Adds a rule on the subscription as defined by the given rule name, filter and action
+   * @param ruleName Name of the rule
+   * @param filter true or false
+   * @param sqlRuleActionExpression Action to perform if the message satisfies the filtering expression
+   */
+  async addBooleanRule(ruleName: string, filter: boolean, sqlRuleActionExpression?: string): Promise<void> {
```

Comment:
I am also debating having a single addRule vs multiple. The single one would look like

`async addRule(ruleName, filter: boolean|SQLFilter|CorrelationFilter, sqlRuleActionExpression?: string): Promise<void>`